### ISSUE TITLE
Templates - Rename "list" (and listi18n) parameter to "options".

### ISSUE THREAD

**Issue by sleroux-keep** [keeps/roda#497]
```
{{~field name="levelOfDetail" type="list" list="{\"Full\": {\"en\":\"Full\", \"pt_PT\":\"Completo\"}, \"Partial\": {\"en\":\"Partial\", \"pt_PT\":\"Parcial\"}, \"Minimal\": {\"en\":\"Minimal\", \"pt_PT\":\"Mínima\"}}" (...) ~}}
```
to 
```
{{~field name="levelOfDetail" type="list" options="{\"Full\": {\"en\":\"Full\", \"pt_PT\":\"Completo\"}, \"Partial\": {\"en\":\"Partial\", \"pt_PT\":\"Parcial\"}, \"Minimal\": {\"en\":\"Minimal\", \"pt_PT\":\"Mínima\"}}" (...) ~}}
```

**Comment by sleroux-keep** [comment]
Fixed at 490527f545073c384afbdc81a759df8537ce4564

Ticket also updated for roda-in (https://github.com/keeps/roda-in/issues/187).

**Comment by luis100** [comment]
checked.
